### DISCUSSION TITLE
How to replace `Get.putAsync` in the version 5 of Getx?

### DISCUSSION THREAD

**Discussion by sumitsharansatsangi** [jonataslaw/getx#3058] [Q&A]
In Getx -version 4 , we have Get.putAsync, what is it's alternative in the Getx 5 version?

**Comment by egortabula** [comment]
> In Getx -version 4 , we have Get.putAsync, what is it's alternative in the Getx 5 version?

Hi! I find this solution. It works for me. 

` await Get.put(AppwriteService()).init();`

Hope, it will help you)
